docs/production-deployment/cloud/service-accounts.mdx

```diff
@@ -135,31 +136,79 @@ tcld service-account delete --service-account-id "e9d87418221548"
-6. Click the Save button located in the bottom left of the screen.
-   A status message is displayed at the bottom right corner of the screen.
+4. Select `Edit` from the menu displayed
+5. Make changes to the Service Account
+
+    - You can change the Service Account's name, description, Account Level Role, and Namespace Permissions
+6. Click the `Save` button located in the bottom left of the screen
+
+    - A status message is displayed at the bottom right corner of the screen
 
 #### Using tcld
 
 Three different commands exist to help users update a Service Account using tcld:
 
-- `tcld service-account update`: To update a Service Account's name or description field
-- `tcld service-account set-account-role`: To update a Service Account's Account Role
-- `tcld service-account set-namespace-permissions`: To update a Service Account's Namespace Permissions
+- `tcld service-account update`: to update a Service Account's name or description field
+- `tcld service-account set-account-role`: to update a Service Account's Account Role
+- `tcld service-account set-namespace-permissions`: to update a Service Account's Namespace Permissions
 
 Example:
 
 ```
-tcld update --id "2f68507677904e09b9bcdbf93380bb95" -d "new description"
+tcld service-account update --id "2f68507677904e09b9bcdbf93380bb95" -d "new description"
 ```
+
+## Namespace Scoped Service Accounts {#scoped}
+
+There is a special type of Service Account, called a Namespace Scoped Service Account, which shares the
+same functionality as the Service Accounts above, but is limited (or scoped) to a single namespace.
+
+In particular, a Namespace Scoped Service Account must _always_ have:
+* An Account Role of `Read`
+* A single Namespace Permission
+
+Note that a Namespace Scoped Service Account can't be re-assigned to a different Namespace after creation,
+but the value of the Namespace Permission can be changed (e.g. from `Read` to `Write`).
+
+Namespace Scoped Service Accounts are useful in situations where you want to ensure a client only has access to a single Namespace (and nothing else).
+
+You can retrieve, update, and delete a Namespace Scoped Service Account using the same process and commands as above, but creation is slightly different.
+
+### Create a Namespace Scoped Service Account
+
+As with regular Service Accounts, Namespace Scoped Service Accounts can be created using Temporal Cloud UI or tcld.
+
+#### Using the Cloud UI
+
+Currently, creating a Namespace Scoped Service Account from the Temporal Cloud UI happens on an individual [Namespace](/cloud/namespaces#manage-namespaces) page.
+If the current Namespace has API key authentication enabled, then there will be a `Generate API Key` button as a banner on the top of the Namespace page or in the `Authentication` section.
+
+By clicking on the `Generate API Key` button, a Namespace Scoped Service Account will be automatically created for the given Namespace and an associated API key will be displayed.
```

Comment:
Do we want to mention that if a scoped service account already exists for this Namespace it will be used by default?

Reply:
ah, true. i can include that too...